src/views/Pools/components/CakeVaultCard/CakeVaultTokenPairImage.tsx

```diff
@@ -1,10 +1,9 @@
 import React from 'react'
 import { TokenPairImage, ImageProps } from '@pancakeswap/uikit'
-import tokens from 'config/constants/tokens'
-import { getAddress } from 'utils/addressHelpers'
+import { mainnetTokens } from 'config/constants/tokens'
 
 const CakeVaultTokenPairImage: React.FC<Omit<ImageProps, 'src'>> = (props) => {
-  const primaryTokenSrc = `/images/tokens/${getAddress(tokens.cake.address)}.svg`
+  const primaryTokenSrc = `/images/tokens/${mainnetTokens.cake.address}.svg`
```

Comment:
Are we always using mainnet?

Reply:
We don't even have testnet cake vault...  or image for testnet cake address.... :(

Reply:
Yea as Cheems points out - sometimes we don't have a testnet equivalent. Sometimes that doesn't really matter (like here - it's just pointing to an svg of the cake icon), and other times it does (because we're missing functionality on testnet).